blog/2020-11-05-HowMuchFasterIsJava15.adoc

```diff
@@ -0,0 +1,105 @@
+= How much faster is Java 15?
+:page-interpolate: true
```

Comment:
@michaltomco Tip: add `:imagesdir: 2020/12` to get rid of the "File doesn't resolve" error and see the images in the Preview pane in IDEA.